src/test/java/com/snowflake/kafka/connector/UtilsTest.java

```diff
@@ -283,6 +284,8 @@ public void testGetExceptionMessage() throws Exception {
   }
 
   @Test
+  @Ignore
+  // OAuth tests are temporary disabled
```

Comment:
Junit's ignore accepts the reason as a parameter. Then this reason can be displayed in IDE and other tools
```java
@ Ignore("OAuth tests are temporary disabled")
```